index.html

```diff
@@ -2731,7 +2739,9 @@ <h3><dfn>Writing or pushing content</dfn></h3>
                   </ul>
                   <dt><a>external type name</a></dt>
                   <ul>
-                    If |record|'s <a>recordType</a> is an <a>external type</a>,
+                    If |record|'s <a>recordType</a> qualifies as an
```

Comment:
Quess qualifies should link somewhere

Reply:
Guess

Reply:
Yeah, that's why I have put 2 TODO's to factor out external type and local type algorithms, but maybe not in this PR, since it's mostly syntactic.

Reply:
Or should it be here?

Reply:
Added local and external type checks. Needs review.